zmanim/hebrew_calendar/jewish_calendar.py

```diff
@@ -9,13 +9,17 @@
 
 
 class JewishCalendar(JewishDate):
-    SIGNIFICANT_DAYS = Enum('SignificantDays', 'erev_rosh_hashana rosh_hashana tzom_gedalyah erev_yom_kippur yom_kippur \
-                                                erev_succos succos chol_hamoed_succos hoshana_rabbah shemini_atzeres simchas_torah \
-                                                chanukah tenth_of_teves tu_beshvat \
-                                                taanis_esther purim shushan_purim purim_katan shushan_purim_katan \
-                                                erev_pesach pesach chol_hamoed_pesach pesach_sheni erev_shavuos shavuos \
-                                                seventeen_of_tammuz tisha_beav tu_beav \
-                                                yom_hashoah yom_hazikaron yom_haatzmaut yom_yerushalayim')
+    SIGNIFICANT_DAYS = Enum(
+        'SignificantDays', 'erev_pesach pesach chol_hamoed_pesach pesach_sheni \
+                            erev_shavuos shavuos seventeen_of_tammuz \
+                            tisha_beav tu_beav erev_rosh_hashana rosh_hashana \
+                            fast_of_gedalyah erev_yom_kippur yom_kippur \
```

Comment:
Finally getting to this :)  

The naming convention for all significant days was taken from the KosherJava project, and I'd prefer not to deviate unless there were a compelling reason to do so.  Also, please note that Lag B'Omer has now been added to the mix, and the spelling has been changed from the 'Lag BaOmer' previously used (in the omer formatting logic).